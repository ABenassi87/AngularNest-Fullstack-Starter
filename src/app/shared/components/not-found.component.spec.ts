import { ComponentFixture, TestBed } from '@angular/core/testing';
import { By } from '@angular/platform-browser';
import { DebugElement } from '@angular/core';

import { NotFoundComponent } from './not-found.component';

describe('NotFoundComponent', () => {
  let fixture: ComponentFixture<NotFoundComponent>;

  beforeEach(() => {
    TestBed.configureTestingModule({
      declarations: [NotFoundComponent]
    });

    fixture = TestBed.createComponent(NotFoundComponent);
  });

  it('renders markup to snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
