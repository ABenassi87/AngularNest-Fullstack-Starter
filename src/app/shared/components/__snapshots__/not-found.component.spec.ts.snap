// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundComponent renders markup to snapshot 1`] = `
<not-found>
  
      <h1
    _ngcontent-c0=""
  >
    404
  </h1>
    
</not-found>
`;
